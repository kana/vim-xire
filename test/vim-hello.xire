; "Hello world" in xire.

(echo "Hello")
(echo 'World)
(echo 142857)

(echo "Hello" 'World 142857)

(echo "'\"\n\r\\")

; vim: filetype=scheme
