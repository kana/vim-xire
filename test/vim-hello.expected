Hello
Error detected while processing ./test/,vim-hello.vim:
line    2:
E121: Undefined variable: World
E15: Invalid expression: World
142857
Hello
line    4:
E121: Undefined variable: World
E15: Invalid expression: World 142857
'"
\


**** Result ****


0 examples, 0 failures
